flink-connector-aws/flink-connector-dynamodb/pom.xml

```diff
@@ -34,6 +34,9 @@ under the License.
 
     <packaging>jar</packaging>
 
+    <properties>
+        <flink.connector.module.config><!-- Required by DynamoDbSinkITCase --> --add-opens=java.base/java.util=ALL-UNNAMED</flink.connector.module.config>
+    </properties>
```

Comment:
`DynamoDbSinkITCase` also requires `java.base/java.lang` to be open
```
[ERROR] org.apache.flink.connector.dynamodb.sink.DynamoDbSinkITCase.testRandomDataSuccessfullyWritten  Time elapsed: 0.045 s  <<< ERROR!
java.lang.reflect.InaccessibleObjectException: Unable to make field private final byte[] java.lang.String.value accessible: module java.base does not "opens java.lang" to unnamed module @365c30cc
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
        at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:178)
        at java.base/java.lang.reflect.Field.setAccessible(Field.java:172)
        at org.apache.flink.api.java.ClosureCleaner.clean(ClosureCleaner.java:106)
        at org.apache.flink.api.java.ClosureCleaner.clean(ClosureCleaner.java:132)
        at org.apache.flink.api.java.ClosureCleaner.clean(ClosureCleaner.java:69)
        at org.apache.flink.streaming.api.environment.StreamExecutionEnvironment.clean(StreamExecutionEnvironment.java:2317)
        at org.apache.flink.streaming.api.datastream.DataStream.clean(DataStream.java:202)
        at org.apache.flink.streaming.api.datastream.DataStream.map(DataStream.java:576)
        at org.apache.flink.connector.dynamodb.sink.DynamoDbSinkITCase.getRandomDataGenerator(DynamoDbSinkITCase.java:230)
        at org.apache.flink.connector.dynamodb.sink.DynamoDbSinkITCase.testRandomDataSuccessfullyWritten(DynamoDbSinkITCase.java:230)
```

Reply:
sorry it seems I missed it....

now fixed
please have a look once you have time